ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -180,13 +193,30 @@ class SignalProducerSpec: QuickSpec {
 		}
 
 		describe("start") {
-			pending("should immediately begin sending events") {
+			it("should immediately begin sending events") {
+				var isSended = false
+				let signalProducer = SignalProducer<Int, NoError>(value: 0)
+				signalProducer.start(next:{ _ -> Void in isSended = true })
+				
+				expect(isSended).to(beTruthy())
 			}
 
-			pending("should send interrupted if disposed") {
+			it("should send interrupted if disposed") {
+				var interrupted = false
+				let (producer, sink) = SignalProducer<Int, NoError>.buffer()
+				let disposable = producer.start(interrupted:{ interrupted = true })
+				
+				expect(interrupted).to(beFalsy())
+				disposable.dispose()
+				expect(interrupted).to(beTruthy())
 			}
 
 			pending("should release sink when disposed") {
+				let (producer, sink) = SignalProducer<Int, NoError>.buffer()
+				let disposable = producer.start()
```

Comment:
This test is actually meant to test the lifecycle of a sink _passed to_ `start()`, not the one returned from `buffer()`.